sdk/eventhub/event-hubs/test/stress/app/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+export interface SnapshotOptions {
```

Comment:
The content of the module doesn't fit with the module name I think.

Reply:
added more to it